src/main/java/org/opentripplanner/graph_builder/module/osm/wpss/AtlantaWayPropertySetSource.java

```diff
@@ -1,8 +1,10 @@
-package org.opentripplanner.graph_builder.module.osm;
+package org.opentripplanner.graph_builder.module.osm.wpss;
```

Comment:
We are not using abbreviated package names, except where the name is an official name of a standard, such as OSM and GTFS.

Reply:
What about calling it `source`?

Reply:
The build-config.json parameter is nowadays called `osmTagMapping`. Would it make sense to rename the package/files to follow that naming convention too?

Reply:
I think so but a quick discussion in today's meeting is probably a good idea.